src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -260,3 +266,18 @@ function genDockerFile(serviceNameAndRelativePath: string, platform: Platform, o
 
     return contents;
 }
+
+async function initializeForDebugging(context: IActionContext, folder: WorkspaceFolder, platformOS: PlatformOS, { artifactName }: Partial<PackageInfo>): Promise<void> {
+    const scaffoldContext: DockerDebugScaffoldContext = {
+        folder: folder,
+        platform: 'netCore',
```

Comment:
>platform [](start = 8, length = 8)

Using "platform" to refer to the programming language/framework (e.g. netCore, node) and "platformOS" to refer to the operating system (e.g. Windows and Linux) seems sub-ideal.

Reply:
How so?

Reply:
Leads to confusion around whether a variable named "platform" means "Linux" or it means "netCore"

Reply:
Well, I don't think there's anywhere in the code where we call a PlatformOS 'platform'; it's always 'os' or 'platformOS'. I'd prefer not to refactor nearly every file in the source just for this.